plugins/CoreAdminHome/templates/trackingCodeGenerator.twig

```diff
@@ -24,6 +24,7 @@
             <a href="#/mobileAppsAndSdks">{{ 'SitesManager_MobileAppsAndSDKs'|translate }}</a>
             <a href="#/trackingApi">{{ 'CoreAdminHome_HttpTrackingApi'|translate }}</a>
             <a href="#/singlePageApplication">{{ 'SitesManager_SiteWithoutDataSinglePageApplication'|translate }}</a>
+            <a href="#/cloudflare">{{ 'SitesManager_SiteWithoutDataCloudflare'|translate}}</a>
```

Comment:
No big deal, but it looks like a space is missing after `translate`.

Reply:
fixed